tests/e2e/framework/kubernetes.go

```diff
@@ -898,6 +905,17 @@ spec:
 `
 )
 
+// Wait for Kubernetes to become active within kubernetesReadinessTimeout period
+// This operation only retreives the pods in the kube-system namespace
+// (TODO) sdake: This may be insufficient for a complete readiness check of Kubernetes
```

Comment:
We could just go through the list and check if all pods are `READY`/`COMPLETED`. Should then be enough, right?

Reply:
yes this would be more comprehensive - I was looking for a fast solution to just verifying kubeapi was operational as the gates were failing when it was not.

Cheers
-steve